### DISCUSSION TITLE
Ecosystem Program Proposal

### DISCUSSION THREAD

**Discussion by casinoyoda** [ringdao-community-guild/discussion#56] [General]
## **1. Introduction**

As a committed member of the DCDAO, focused on the growth and advancement of the Darwinia Network, I propose the introduction of two strategic positions within our community: an Integration Specialist and a Data Analyst. This initiative is rooted in the belief that these roles are integral to enhancing our network's capabilities

## **2. Program Objectives**

The primary objective of this program is to strengthen the Darwinia Network's infrastructure and community engagement. This initiative aims to:

1. **Enhance Technical Integration and Efficiency:** we aim to streamline the integration of various critical services and platforms within the Darwinia ecosystem, such as Msgport, Subscan, Helix, Multisig wallet etc.
2. **Deepen Data Analysis and Community Insight:** The addition of a Data Analyst will enable us to leverage data more effectively, gaining deeper insights into community trends and needs.

I will ensure that the two selected participants receive ample support to effectively commence their roles. Additionally, they will be actively involved in DCDAO discussions to directly capture and address the community's needs and expectations.

### Integration Specialist

**Responsibilities**:

- Oversee and manage Darwinia integration projects on a part-time basis, coordinating closely with external systems.
- Collaborate with technical teams to address and solve integration challenges, contributing to the ecosystem's development in a flexible capacity.
- The current projects demanding integration involve a range of critical services and providers in the ecosystem, including Indexers such as TheGraph, Subquery, and Subsquid; Developer tools; RPC (Remote Procedure Call) providers; various Data Providers like Coingecko; and Wallet Providers, among others..

**Qualifications**:

- Proven experience in system integration, ideally in a part-time or project-based context.
- Technical proficiency in relevant programming languages and tools, adaptable to part-time work schedules.

### Data Analyst

**Responsibilities**:

- Collaboration with Data Platforms:
    - Work part-time with Subscan Open Platform and other key data platforms, focusing on data collection, interpretation, and analysis.
    - Ensure effective data integration and exchange between DCDAO and these platforms in a flexible working arrangement.
- Community-Driven Initiatives:
    - Employ data analysis skills to understand and support community needs and engagement, tailored to a part-time role.
    - Provide analytical support to community projects and initiatives on a part-time basis.

**Qualifications**:

- Strong analytical skills, ideally suited for part-time involvement.
- Proficiency in data analysis tools and methodologies, capable of delivering insights in a part-time capacity.

## **4. Application and Selection Process**

1. **Submission via Discussion or Email:**
    - Interested candidates are invited to submit their applications in the [[discussion channel of DCDAO](https://github.com/orgs/dcdao/discussions)](https://github.com/orgs/dcdao/discussions) or via the email. This submission should serve as an introduction, providing a comprehensive overview of their qualifications, experience, and motivation for applying.
    - Candidates should clearly articulate how their skills, experiences, and interests align with the specific demands of the roles they are applying for.
2. **Review by DCDAO Steering Committee:**
    - The DCDAO steering committee will undertake a thorough review of all applications. This committee, composed of experienced members from various domains within the network, will assess each candidate based on their qualifications, experience, and overall fit for the role.
    - The review process will focus on identifying candidates who not only possess the necessary technical skills but also demonstrate a strong alignment with the Darwinia Network’s values and community-driven approach.
3. **Majority Vote Decision:**
    - After careful consideration, the steering committee will select suitable candidates through a majority vote.
    - The chosen candidates will then be notified and provided with further details regarding their roles and the next steps in the onboarding process.

## **5. Funding and Support**

To support the crucial roles of the Integration Specialist and Data Analyst, we are offering a dedicated funding plan. Qualified candidates will be granted a funding period of three months.

During this period, each participant will receive up to $1,500 per month. This financial support is designed to facilitate their focus and dedication to their roles, ensuring they have the resources needed to effectively contribute to the Darwinia Network.

The funds will be distributed from the DCDAO’s multi-signature wallet, This distribution will occur monthly, ensuring a steady and reliable financial resource for the participants throughout their three-month tenure.

## **6. Expected Outcomes**

Each participant is required to submit a report every month. This report should encompass their activities, progress, and any challenges encountered during their tenure. These reports serve multiple purposes: they provide transparency, allow for ongoing assessment of the participant's contributions, and offer insights into how their work is influencing the Darwinia Network.

**Comment by casinoyoda** [comment]
From this discussion: https://github.com/orgs/dcdao/discussions/55

**Comment by hackfisher** [comment]
LGTM

**Comment by Laki389** [comment]
I hope that suitable candidates will be chosen, with suitable qualifications.

If there are no candidates with satisfactory qualifications, I would suggest not to select candidates at all costs from the Darwinia/DCDAO community, I would put quality before affiliation,
so perhaps the Darwinia twitter profile could be used as an advertisement, a call for interested candidates, Polkadot's official Discord server or perhaps specialized websites with job advertisements in the WEB3 field.

I hope for quality individuals to contribute to the development and progress of Darwinia and DCDAO.
